src/elementPlugins/dropdownChoice/src/ApplicationView.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react'
+import { Dropdown, Header, Message } from 'semantic-ui-react'
+import { ApplicationViewProps } from '../../types'
+
+const ApplicationView: React.FC<ApplicationViewProps> = ({
+  templateElement,
+  onUpdate,
+  initialValue,
+  isEditable,
+  currentResponse,
+  validationState,
+  onSave,
+}) => {
+  const {
+    label,
+    description,
+    placeholder,
+    options,
+    default: defaultIndex,
+  } = templateElement?.parameters
+
+  const [value, setValue] = useState(initialValue ? initialValue?.text : options[defaultIndex])
+
+  useEffect(() => {
+    onUpdate(value)
+    if (!initialValue) onSave({ text: value, optionIndex: options.indexOf(value) })
```

Comment:
hmm any reason to save on start up ? Wouldn't it be exact same value as already stored ? Or is it for validation ?

Reply:
It's because the DropDown can have a default selection -- in which case we need to save this value immediately, as it's already got a valid response before the user has even done anything.

The shortText plugin currently doesn't handle a default value, but if it did, we'd need to do the same for that too.

Although, I've now changed it to actually check there is a default value provided before saving:
```
if (!initialValue && defaultIndex) onSave({ text: value, optionIndex: options.indexOf(value) })
```